vanilla-tests/src/main/java/fixtures/additionalproperties/Pets.java

```diff
@@ -147,6 +147,20 @@ public Mono<PetAPTrue> createAPTrueAsync(PetAPTrue createParameters) {
         return createAPTrueWithResponseAsync(createParameters).flatMap(res -> Mono.justOrEmpty(res.getValue()));
     }
 
+    /**
+     * Create a Pet which contains more properties than what is defined.
+     *
+     * @param createParameters The createParameters parameter.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response body along with {@link Response}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<PetAPTrue> createAPTrueWithResponse(PetAPTrue createParameters) {
+        return createAPTrueWithResponseAsync(createParameters).block();
+    }
```

Comment:
since there is too many this change to generated tests (I am fine if it is only for vanilla), I didn't check all tests.

Notify us if any changes to tests is noteworthy.